src/app/feedback/feedback.component.html

```diff
@@ -0,0 +1,42 @@
+<h1 i18n>Add Feedback</h1>
+<form (ngSubmit)="submitfeedback()">
+  <div class="row form-group">
+    <label class="col-4" i18n>Is Your feedback Urgent?</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+      <input class="form-check-input" type="radio" value="true" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> Yes
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+      <input class="form-check-input" type="radio" value="false" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> No
+      </label>
+    </div>
+  </div>
+  <div class="row form-group">
+    <label class="col-3" i18n>Type of Feedback</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+      <input class="form-check-input" type="radio" value="Suggestion" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Suggestion
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3" i18n>
+      <input class="form-check-input" type="radio" value="Bug" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Bug
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3" i18n>
+      <input class="form-check-input" type="radio" value="Question" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Question
+     </label>
+   </div>
+ </div>
+ <div class="form-group">
+    <h5 i18n>Feedback</h5>
+    <textarea type="text" name="feedbackMsg"[(ngModel)]="feedback.feedbackMsg" [rows]="7" [cols]="50" required="true"></textarea>
+    <div *ngIf="msgForUsr" class="alert alert-danger" i18n>{{msgForUsr}}</div>
```

Comment:
This is an example of something that we (I started it) need to refactor to get the i18n translation to work.  Let's include one alert here, but take a look at these resources to improve this message:

* https://angular.io/guide/i18n#select-among-alternative-text-messages
  First rather than having the message itself in a variable, you should have a `message` variable that can be `error`, 
`success`, or `invalid` and implement the **Select among alternative text messages** section.

* https://angular.io/api/common/NgClass
  Second, we don't need three `div`s here for different alert types.  You can use an `[ngClass]` directive to set the class to `alert-success` if the `message` is `success` or `alert-danger` otherwise (with the ngIf set to `message` so it's not showing when there is no message)

Let me know if you have any questions or if you need any help with this code

Reply:
@paulbert I initially thought to implement that way. I will look at the resources and update the alert accordingly. Thanks!